src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java

```diff
@@ -282,7 +284,7 @@ public void testSelectBlob(TestContext should) {
       .onSuccess(rows -> {
         should.assertEquals(1, rows.size());
         for (Row row : rows) {
-          should.assertNull(row.getString(0));
+          //should.assertNull(row.getString(0));
```

Comment:
@vietj this throw `ClassCastException` now, weren't we saying that toString is always valid?

Reply:
now this has changed recently.

Reply:
now it should throw CCE. `null` shall be returned when it's actually null SQL value

Reply:
@pmlopes ping